app/code/Magento/MediaGalleryCatalogUi/Test/Mftf/Test/AdminMediaGalleryCatalogUiVerifyUsedInLinkProductGridTest.xml

```diff
@@ -29,6 +29,8 @@
         <after>
             <magentoCLI command="config:set cms/wysiwyg/enabled disabled" stepKey="disableWYSIWYG"/>
             <actionGroup ref="ResetAdminDataGridToDefaultViewActionGroup" stepKey="resetAdminDataGridToDefaultView"/>
+            <amOnPage url="{{AdminProductIndexPage.url}}" stepKey="navigateToProductIndex"/>
```

Comment:
Hi @joweecaquicla. What do you think about using an existing action group instead of a direct call? As the result, we will rely more on action groups which is part of the good practice 

https://github.com/magento/magento2/blob/0262f1428561f5c329a50bfbd62714fa58e175be/app/code/Magento/Catalog/Test/Mftf/ActionGroup/GoToProductCatalogPageActionGroup.xml#L9